doc/ipython-notebooks/classification/Classification.ipynb

```diff
@@ -87,11 +88,11 @@
    },
    "outputs": [],
    "source": [
-    "shogun_feats_linear = RealFeatures(CSVFile('../../../data/toy/classifier_binary_2d_linear_features_train.dat'))\n",
-    "shogun_labels_linear = BinaryLabels(CSVFile('../../../data/toy/classifier_binary_2d_linear_labels_train.dat'))\n",
+    "shogun_feats_linear = RealFeatures(CSVFile(SHOGUN_DATA_DIR + 'toy/classifier_binary_2d_linear_features_train.dat'))\n",
```

Comment:
Thanks for these comments. Just finished refactored and squashed all fixup commits:)